internal/server/kong/ws/config/version_compatibility.go

```diff
@@ -570,6 +580,12 @@ func (vc *WSVersionCompatibility) processCoreEntityUpdates(payload string,
 				if gjson.Get(updatedRaw, conditionField).Exists() {
 					for _, fieldUpdate := range update.Updates {
 						conditionUpdate := fieldUpdate.Field
+						// Ensure the original field is not empty if specified; do not overwrite
+						if fieldUpdate.FieldMustBeEmpty && !valueIsEmpty(gjson.Get(updatedRaw, conditionUpdate)) {
+							// Since this is a copy function from another field and the current field is already
+							// configured the entire field update process should short circuit
+							break
```

Comment:
I guess this should be `continue` like in line 398?

Reply:
Switching from `break` to `continue` in both of these should update the comments (and update the doc string for `FieldMustBeEmpty`.

Reply:
Thanks for catching!